auto_gptq/modeling/_base.py

```diff
@@ -405,6 +405,11 @@ def forward(self, inp=None, **kwargs):
         inside_layer_modules = self.inside_layer_modules
         if not self.quantize_config.true_sequential:
             inside_layer_modules = [sum(inside_layer_modules, [])]
+
+        if hasattr(self.model.config, "num_experts"):
+            inside_layer_modules = get_moe_inside_layer_modules(self.inside_layer_modules,
+                                                                self.model.config.num_experts)
```

Comment:
Why is `self.inside_layer_modules` not enough?

Reply:
We cannot write the names of all the parameters in our MoE model because we may have more experts. Besides, we may have different numbers of experts for different model sizes.

Reply:
I agree it is a bit clunky but you can just add like the highest amount of experts you may have and the models with less will work too.

Reply:
> I agree it is a bit clunky but you can just add like the highest amount of experts you may have and the models with less will work too.

It will work, but I think the current codes look much better than writing hundreds of parameter names in the modeling file.

Reply:
I don't disagree, but I would prefer a separate PR for for something like this that others could rely on. I know that is just for slightly easier git history. I just prefer PRs to stay focused and do the minimum changes required. Feel free to ignore me, it's not that important.